lab_3_ann_retriever/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "target_score": 0
+    "target_score": 6
```

Comment:
`save` and `load` for 10? I believe you can take 10!

Reply:
why is this score so strange? I'm waiting 10!

Reply:
Sorry, but 8 is my maximum